packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -57,114 +58,131 @@ describe('importCSV', function () {
   for (const filepath of Object.values(fixtures.csv)) {
     const basename = path.basename(filepath);
 
-    it(`imports ${basename}`, async function () {
-      const typeResult = await guessFileType({
-        input: fs.createReadStream(filepath),
-      });
-      assert(typeResult.type === 'csv');
-      const csvDelimiter = typeResult.csvDelimiter;
-
-      const analyzeResult = await analyzeCSVFields({
-        input: fs.createReadStream(filepath),
-        delimiter: csvDelimiter,
-        ignoreEmptyStrings: true,
-      });
-
-      const abortController = new AbortController();
-      const progressCallback = sinon.spy();
-
-      const ns = 'db.col';
+    for (const lineEnding of ['unix', 'windows']) {
```

Comment:
Our fixtures all have unix line endings. Here I loop through them twice and the second time we also replace all the line endings with windows ones and make sure it detects the correct line ending and that it inserts the exact same docs in both cases.